packages/core/src/tools/create-file-stasher.js

```diff
@@ -295,6 +306,8 @@ const createFileStasher = (input) => {
       filename: _filename,
     } = await resolveToBufferStringStream(responseOrData);
 
+    isUploadMaxSizeExceeded(streamOrData, knownLength || length);
```

Comment:
`knownLength || length` is repeated. Can we assign it to a variable:

```suggestion
    const finalLength = knownLength || length;
    isUploadMaxSizeExceeded(streamOrData, finalLength);
```

and reuse it below too?

```js
    return uploader(
      signedPostData,
      streamOrData,
      finalLength,   // <--
      filename || _filename,
      contentType || _contentType
    );
```

Reply:
Done